safe/gui/tools/metadata_converter_dialog.py

```diff
@@ -318,3 +322,23 @@ def show_current_metadata(self):
         content_html = keywords.to_message().to_html()
         full_html = html_header() + content_html + html_footer()
         self.metadata_preview_web_view.setHtml(full_html)
+
+    # Adapted from https://stackoverflow.com/a/43126946/1198772
+    def resizeEvent(self, event):
+        """Emit custom signal when the window is re-sized.
+
+        :param event: The re-sized event.
+        :type event: QResizeEvent
+        """
+        self.resized.emit()
+        return super(MetadataConverterDialog, self).resizeEvent(event)
+
+    def after_resize(self):
+        """Method after resizing the window."""
+        # TODO(IS) this is a QT bug :
+        # https://stackoverflow.com/q/25644026/1198772
+        # It's fixed in QT 5.4 https://bugreports.qt.io/browse/QTBUG-37673
+        # If you port it to QGIS 3 (which use QT 5.9, please remove this
```

Comment:
QGIS Minimum QT version is 5.2 https://github.com/qgis/QGIS/blob/master/CMakeLists.txt#L277